docs/developing.md

```diff
@@ -266,6 +267,74 @@ To recreate *_vendor_* directory, you can run the following command:
 make vendor
 ```
 
+### Vendoring the OpenShift Installer
+
+For various reasons, Hive vendors the OpenShift Installer. The OpenShift installer brings in quite a few dependencies, so it is important to know the general flow of how to vendor the latest version of the OpenShift Installer.
+
+Things to note:
+* Hive vendors the installer from `@master`, NOT from `@latest`. For go modules, `@latest` means the latest github release, which for the installer is not up to date.
+* The `go.mod` file contains a section called `replace`. The purpose of this section is to force go modules to use a specific version of that dependency.
+* `replace` directives may need to be copied from the OpenShift Installer or possibly other Hive dependencies. In other words, any dependency may need to be pinned to a specific version.
+* `go mod tidy` is used to add (download) missing dependencies and to remove unused modules.
+* `go mod vendor` is used to copy dependent modules to the vendor directory.
+
+
+The following is a basic flow for vendoring the latest OpenShift Installer. Updating Go modules can sometimes be complex, so it is likely that the flow below will not encompass everything needed to vendor the latest OpenShift Installer. If more steps are needed, please document them here so that the Hive team will know other possible things that need to be done.
+
+Basic flow:
+* Vendor the latest installer. In order to vendor the installer, you must use `go get` with a specific source package. For instance:
+```
+go get github.com/openshift/installer/pkg/asset/machines/vsphere@master
```

Comment:
Why do you get a source package instead of `go get github.com/openshift/installer@master`?

Reply:
Ah, yes. That is because you should be using the `-d` flag since you do not want to actually install the module. You should also probably use the `-u` flag as well.